models/dbt_queries.sql

```diff
@@ -1,6 +1,6 @@
 {{ config(
     materialized='incremental',
-    unique_key=['query_id', 'start_time']
+    unique_key=['query_id', 'account_locator', 'start_time'] if var('uses_org_view', false) else ['query_id', 'start_time']
```

Comment:
I would highly recommend using `account_name` as the identifier column across the board - Snowflake have deprecated account_locator 😭

Reply:
```suggestion
    unique_key=with_optional_account_name(['query_id', 'start_time'])
```
maybe add a macro to do this in a DRY way, appending `account_name`?